linux_kernel_modules/mangoh/mangoh_yellow.c

```diff
@@ -220,8 +222,19 @@ static void eeprom_setup(struct memory_accessor *mem_acc, void *context)
 		goto done;
 	}
 
-	if (strcmp(data, dv3_string) == 0)
+	for (i = 0; i < eeprom_size; i++) {
+		if (data[i] != 0xFF) {
+			eeprom_blank = false;
+			break;
+		}
+	}
+
+	if (eeprom_blank) {
+		pr_warn("EEPROM is empty - assuming board is a mangOH Yellow DV3\n");
+		mangoh_yellow_driver_data.board_rev = BOARD_REV_DV3;
```

Comment:
I think we should assume BOARD_REV_NEWEST if the EEPROM is blank, where BOARD_REV_NEWEST = BOARD_REV_DV3 for now.

Reply:
Good suggestion.

Reply:
done